src/Attachment.ts

```diff
@@ -101,6 +101,11 @@ export default class Attachment {
       this.properties.set('thumbnail', thumbnail);
    }
 
+   public setGenericThumbnail()
```

Comment:
We probably don't need this method, because we will always use the same image as "generic thumbnail". This means we can easily test if it's used or not.

Reply:
removed... and set generic image as default..
this would be overwritten if used otherwise with an other thumbnail...